dev/bots/test.dart

```diff
@@ -176,6 +178,58 @@ Future<bq.BigqueryApi> _getBigqueryApi() async {
   }
 }
 
+// Partition tool tests into two groups, see explanation on `_runToolCoverage`.
```

Comment:
It seems like we might be able to use the same logic as what I'm attempting in https://github.com/flutter/flutter/pull/34877

Reply:
I'm not quite sure - the coverage collection uses a test runner that I wrote, since it isn't supported by package:test or build_runner

Reply:
Ahh ok.